raiden_contracts/contract_manager.py

```diff
@@ -47,21 +49,26 @@ class ContractManagerVerificationError(RuntimeError):
 
 
 class ContractManager:
-    def __init__(self, path: Union[Path, Dict[str, Path]]) -> None:
+    def __init__(self, flavor: Flavor, path: Union[Path, Dict[str, Path]]) -> None:
         """Params:
             path: either path to a precompiled contract JSON file, or a list of
                 directories which contain solidity files to compile
         """
+        self.flavor = flavor
         self.contracts_source_dirs: Optional[Dict[str, Path]] = None
         self.contracts: Dict[str, Dict] = dict()
         self.overall_checksum = None
         self.contracts_checksums: Optional[Dict[str, str]] = None
         if isinstance(path, dict):
             self.contracts_source_dirs = path
-            self.contracts_version = CONTRACTS_VERSION
+            self.plain_version: PlainVersion = CONTRACTS_VERSION
+            self.contracts_version: FlavoredVersion = contract_version_string(
+                flavor,
```

Comment:
again style consistency with Raiden. Try to use kwargs.

Reply:
I'll fix these all over in this PR.

Reply:
I guess that's not only within in this PR, so I created an issue https://github.com/raiden-network/raiden-contracts/issues/643 .